quadrat/template-parts/header.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Displays the header. 
+ * Needed until we have a consistent mechanism to render a navigation between FSE and classic menus.
+ * 
+ * @package Quadrat
+ * @since 1.0.0
+ * 
+ */
+$blog_info    = get_bloginfo( 'name' );
+$description  = get_bloginfo( 'description', 'display' );
+$has_primary_nav       = has_nav_menu( 'primary' );
+$has_primary_nav_items = wp_nav_menu(
+	array(
+		'theme_location' => 'primary',
+		'fallback_cb'    => false,
+		'echo'           => false,
+	)
+);
+?>
+
+<header id="masthead" class="site-header" role="banner">
+	<div class="site-branding">
+		<?php if ( has_custom_logo() ) : ?>
+			<div class="site-logo"><?php the_custom_logo(); ?></div>
```

Comment:
I think we could reduce this file to only use php for mostly just the navigation. We could replace some of the php using blocks like in this case we could simply do `<?php echo do_blocks( '<!-- wp:site-logo {"align":"center","width":128} /-->' ); ?>` I tested locally and that seemed to work fine and was editable via the customizer